packages/dropdowns.next/src/views/combobox/StyledTagsButton.ts

```diff
@@ -36,7 +36,9 @@ export const StyledTagsButton = styled(StyledValue as 'button').attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledTagsButtonProps>`
+  display: inline-flex;
   flex: 0 1 auto; /* [1] */
+  align-items: center;
```

Comment:
fixes a pixel off alignment issue observed between this button and value/input text